opentracing-noop/src/main/java/io/opentracing/NoopTracer.java

```diff
@@ -15,19 +15,25 @@
 
 import io.opentracing.propagation.Format;
 
-public final class NoopTracer implements Tracer {
+public interface NoopTracer extends Tracer {
+    static NoopTracer create() {
+        return NoopTracerImpl.INSTANCE;
+    }
```

Comment:
to make this Java 1.7 compatible, we could just move the constant from L25 here.